src/components/Asset.vue

```diff
@@ -240,9 +240,17 @@ export default {
         text.push(this.from);
       }
       return text.join(' ');
+    },
+    openAssetsUids () {
+      return this.$store.state.stateQueryParameters.assets;
```

Comment:
This config should be passed through via props as you may have two Asset(s) components on a page. The origin should be the components in views. The reason is that you may run into conflicts with IDs, if they have the same keys/ids.